### ISSUE TITLE
Sun Ray Installation doesn't work anymore!

### ISSUE THREAD

**Issue by WifiRouterYT** [OpenIndiana/oi-docs#266]
Following the instructions over at https://docs.openindiana.org/handbook/sunray/#install, I can no longer get past the installing IPS packages step as I get the following error:
```
root@openindiana:/export/home/wifi/srs_5.4.0.0-Solaris_11plus.i386# pkg install SUNWut-srss SUNWut-srwc SUNWuti
Creating Plan (Running solver): /
pkg install: No matching version of SUNWut-srwc can be installed:
  Reject:  pkg://sunray/SUNWut-srwc@5.4.0.0.44-0.0:20130220T220126Z
  Reason:  No version matching 'require' dependency SUNWuttsc-kiosk@2.6.0.0.42-0.0 can be installed
    ----------------------------------------
    Reject:  pkg://sunray/SUNWuttsc-kiosk@2.6.0.0.42-0.0:20130220T220115Z
    Reason:  No version matching 'require' dependency SUNWkio can be installed
      ----------------------------------------
      Reject:  pkg://sunray/SUNWkio@4.5.0.0.44-0.0:20130220T220654Z
      Reason:  No version matching 'require' dependency SUNWkio-libs can be installed
        ----------------------------------------
        Reject:  pkg://sunray/SUNWkio-libs@4.5.0.0.44-0.0:20130220T220658Z
        Reason:  All acceptable versions of 'require' dependency on compatibility/packages/SUNWxwplt are obsolete
        ----------------------------------------
      ----------------------------------------
    ----------------------------------------
No matching version of SUNWut-srss can be installed:
  Reject:  pkg://sunray/SUNWut-srss@5.4.0.0.44-0.0:20130220T220731Z
  Reason:  No version matching 'require' dependency SUNWckio@4.5.0.0.3-0.0 can be installed
    ----------------------------------------
    Reject:  pkg://sunray/SUNWckio@4.5.0.0.3-0.0:20130220T220611Z
    Reason:  No version matching 'require' dependency SUNWkio can be installed
    ----------------------------------------
root@openindiana:/export/home/wifi/srs_5.4.0.0-Solaris_11plus.i386# pkg list -a compatibility/packages/SUNWxwplt
NAME (PUBLISHER)                                  VERSION                    IFO
compatibility/packages/SUNWxwplt                  0.5.11-2020.0.1.3          --o
root@openindiana:/export/home/wifi/srs_5.4.0.0-Solaris_11plus.i386#
```

funny enough, i tried this yesterday and i got an error with the library/motif package being obsolete instead. Is it no longer possible to install the Sun Ray Software on OpenIndiana? I'd love to revive an old setup I got off Craiglist for free (a beautiful Sun SunFire V20Z and 4 Sun Ray 3's)

**Comment by merclangrat** [comment]
There is a fix, have you tried with it: https://github.com/OpenIndiana/oi-docs/commit/d258a7854c9a4567c51b89b49a4bb12692ce49c2

Actually, I could install and run Sun Ray software (I have OI/x86 in the VM and Sun Ray 100) with a bit of "magic", OI/Sol11 package system is a bit weird with its "incorporations" (it was much easier in Sol10!). I used `pkg update -v` to update the system, otherwise I even couldn't install mate as the docu says...

What else I did - I'll create an issue or even a PR (shortly: I had to enable tftp, put the firmware there and run `utfwadm` to create .parms file, my Sun Ray didn't want to start without it)

**Comment by WifiRouterYT** [comment]
Yes, I believe that fix was put in after I emailed the guy who wrote the sun ray docs and we got it working. It runs perfectly on real Open Indiana hardware now!

**Comment by JMadgwick** [comment]
Fixed in #268
